epictrack-api/src/api/services/staff.py

```diff
@@ -96,12 +96,12 @@ def find_by_id(cls, _id):
     @classmethod
     def check_existence(cls, email, staff_id=None):
         """Checks if a staff exists with given email address"""
-        return Staff.check_existence(email, staff_id)
+        return Staff.check_existence(email.lower(), staff_id)
 
     @classmethod
     def find_by_email(cls, email):
         """Find staff by email address"""
-        return Staff.find_by_email(email)
+        return Staff.find_by_email(email.lower())
```

Comment:
Is it good to have this convertion at the schema level like the way you did it in the POST